src/components/Form.css

```diff
@@ -0,0 +1,211 @@
+#map {
```

Comment:
Do you have to styles this with a `.css` file? Is it not possible to use styled-components (for consistency reasons)?

Reply:
Well probably I could, yes, I just thought it is easier to ready for code reviews if I put these in an extra css file cause it's about 200 lines of code 😅
Plus these are basically the default values from mapbox. I just changed some values so the styling fits to my app

Reply:
Alright :)